superduperdb/components/listener.py

```diff
@@ -50,7 +52,15 @@ def on_create(self, db: Datalayer) -> None:
 
         # Start cdc service if enabled
         if self.select is not None and self.active:
-            db.cdc.add(self.select.table_or_collection)
+            if CFG.mode == 'production':
```

Comment:
Should we need to use Enum instead of string set? Or xxx.is_prodcution

Reply:
we can yes